### PR TITLE
*: Support show index from table

### PR DESCRIPTION
**Pull request by shenli**
Sequel Pro needs "show index from table" syntax.

### CODE REVIEW

File: plan/plans/show.go [hunk 1]
```diff
@@ -676,3 +685,39 @@ func (s *ShowPlan) fetchShowTriggers(ctx context.Context) error {
...
+		return errors.Trace(err)
+	}
+	for _, idx := range tb.Indices() {
+		for i, col := range idx.Columns {
+			nonUniq := 1
+			if idx.Unique {
+				nonUniq = 0
+			}
+			var subPart interface{}
+			if col.Length != -1 {
```
Comment (ngaut) [inline plan/plans/show.go]: When will Col.Length be -1  ?

Comment (shenli) [inline plan/plans/show.go]: When you do not specify index column prefix length, it  will be -1.
https://github.com/pingcap/tidb/blob/master/parser/parser.y#L4099

Comment (ngaut) [inline plan/plans/show.go]: Should we define it as constant?

Comment (shenli) [inline plan/plans/show.go]: We have a const named UnspecifiedLength. I will use it.

**Comment by shenli** [comment]
PTAL

File: plan/plans/show.go [hunk 2]
```diff
@@ -676,3 +685,39 @@ func (s *ShowPlan) fetchShowTriggers(ctx context.Context) error {
...
+				nonUniq = 0
+			}
+			var subPart interface{}
+			if col.Length != types.UnspecifiedLength {
+				subPart = col.Length
+			}
+			data := []interface{}{
+				tb.TableName().O, // Table
+				nonUniq,          // Non_unique
+				idx.Name.O,       //Key_name
```
Comment (ngaut) [inline plan/plans/show.go]: Keep a blank after //

File: plan/plans/show.go [hunk 3]
```diff
@@ -676,3 +685,39 @@ func (s *ShowPlan) fetchShowTriggers(ctx context.Context) error {
...
+				i + 1,            // Seq_in_index
+				col.Name.O,       // Column_name
+				"utf8_bin",       // Colation
+				0,                //
```
Comment (ngaut) [inline plan/plans/show.go]: Any comments for 0 ?

File: plan/plans/show.go [hunk 4]
```diff
@@ -676,3 +685,39 @@ func (s *ShowPlan) fetchShowTriggers(ctx context.Context) error {
...
+				subPart,          // Sub_part
+				nil,              // Packed
+				"YES",            // Null
+				"BTREE",          //Index_type
```
Comment (ngaut) [inline plan/plans/show.go]: Keep a blank after //

**Comment by coocood** [comment]
LGTM

File: plan/plans/show.go (hunk 4 above)
Comment (shenli) [inline plan/plans/show.go]: PTAL

**Comment by ngaut** [comment]
LGTM

**Pull request merged**